integration-tests/suites/base.go

```diff
@@ -119,20 +124,30 @@ func (s *IntegrationTestSuiteBase) StopCollector() {
 // one if it is nil. This function can be used to get the object before
 // the container is launched, so that Collector settings can be adjusted
 // by individual test suites
-func (s *IntegrationTestSuiteBase) Collector() *common.CollectorManager {
+func (s *IntegrationTestSuiteBase) Collector() (collector_manager.CollectorManager, error) {
```

Comment:
Same as the Executor function - we should handle the errors internally

Reply:
Actually, making it so `Executor()` cannot fail, automatically makes it so this method cannot fail either.